components/date-picker/wrapPicker.tsx

```diff
@@ -1,10 +1,13 @@
 import * as React from 'react';
+import { polyfill } from 'react-lifecycles-compat';
 import TimePickerPanel from 'rc-time-picker/lib/Panel';
 import classNames from 'classnames';
+import * as moment from 'moment';
```

Comment:
why `import * as moment from 'moment'` ? This will return a Module。

![image](https://user-images.githubusercontent.com/1179603/56178175-df738800-6033-11e9-89e3-3b761ae6bdda.png)

Reply:
https://github.com/ant-design/ant-design/pull/9502